ACE/ace/Task_T.h

```diff
@@ -1,197 +1,197 @@
-   */
-  int reply (ACE_Message_Block *mb, ACE_Time_Value *tv = 0);
-
-  /**
-   * Transfer message to the adjacent ACE_Task in a ACE_Stream.  Note
-   * that @a timeout uses <{absolute}> time rather than <{relative}>
-   * time.
-   */
-  int put_next (ACE_Message_Block *msg, ACE_Time_Value *timeout = 0);
-
-  // = ACE_Task utility routines to identify names et al.
-  /// Return the name of the enclosing Module if there's one associated
-  /// with the Task, else returns 0.
-  const ACE_TCHAR *name (void) const;
-
-  // = Pointers to next ACE_Task_Base (if ACE is part of an ACE_Stream).
-  /// Get next Task pointer.
-  ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *next (void);
-
-  /// Set next Task pointer.
-  void next (ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *);
-
-  /// Return the Task's sibling if there's one associated with the
-  /// Task's Module, else returns 0.
-  ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *sibling (void);
-
-  /// Return the Task's Module if there is one, else returns 0.
-  ACE_Module<ACE_SYNCH_USE, TIME_POLICY> *module (void) const;
-
-  /**
-   * Flush the task's queue, i.e., free all of the enqueued
-   * message blocks and unblocks any threads waiting on the queue.
-   * Note that if this conflicts with the C++ iostream <flush>
-   * function, just rewrite the iostream function as ::<flush>.
-   */
-  int flush (u_long flag = ACE_Task_Flags::ACE_FLUSHALL);
-
-  // = Special routines corresponding to certain message types.
-
-  /// Manipulate watermarks.
-  void water_marks (ACE_IO_Cntl_Msg::ACE_IO_Cntl_Cmds, size_t);
-
-  /// Queue of messages on the ACE_Task..
-  ACE_Message_Queue<ACE_SYNCH_USE, TIME_POLICY> *msg_queue_;
-
-  /// true if should delete Message_Queue, false otherwise.
-  bool delete_msg_queue_;
-
-  /// Back-pointer to the enclosing module.
-  ACE_Module<ACE_SYNCH_USE, TIME_POLICY> *mod_;
-
-  /// Pointer to adjacent ACE_Task.
-  ACE_Task<ACE_SYNCH_USE, TIME_POLICY> *next_;
-
-  /// Dump the state of an object.
-  void dump (void) const;
-
-  /// Declare the dynamic allocation hooks.
-  ACE_ALLOC_HOOK_DECLARE;
-
-private:
-
-  // = Disallow these operations.
-  ACE_UNIMPLEMENTED_FUNC (void operator= (const ACE_Task<ACE_SYNCH_USE, TIME_POLICY> &))
-  ACE_UNIMPLEMENTED_FUNC (ACE_Task (const ACE_Task<ACE_SYNCH_USE, TIME_POLICY> &))
-};
-
-#if defined ACE_HAS_EXPLICIT_TEMPLATE_INSTANTIATION_EXPORT
-template class ACE_Export ACE_Task<ACE_MT_SYNCH>;
-template class ACE_Export ACE_Task<ACE_NULL_SYNCH>;
-#endif /* ACE_HAS_EXPLICIT_TEMPLATE_INSTANTIATION_EXPORT */
-
-ACE_END_VERSIONED_NAMESPACE_DECL
-
-#if defined (__ACE_INLINE__)
-#include "ace/Task_T.inl"
-#endif /* __ACE_INLINE__ */
-
-#if defined (ACE_TEMPLATES_REQUIRE_SOURCE)
-#include "ace/Task_T.cpp"
-#endif /* ACE_TEMPLATES_REQUIRE_SOURCE */
-
-#if defined (ACE_TEMPLATES_REQUIRE_PRAGMA)
-#pragma implementation ("Task_T.cpp")
-#endif /* ACE_TEMPLATES_REQUIRE_PRAGMA */
-
-#include /**/ "ace/post.h"
-#endif /* ACE_TASK_T_H */
+// -*- C++ -*-

```

Comment:
I don't know. These changes are the result of auto-merging upstream. Please disregard commits
'd945d56' and '9bc5543'.